addons/website/static/src/scss/secondary_variables.scss

```diff
@@ -50,6 +50,14 @@ $o-website-values-palette-number: length($o-website-values-palettes);
     @return map-get($o-website-values, $key);
 }
 
+@if o-website-value('google-fonts') {
+    $o-theme-font-names: join($o-theme-font-names, o-website-value('google-fonts'));
+    @each $i in o-website-value('google-fonts') {
+        $o-theme-fonts: append($o-theme-fonts, ('' + $i + '', sans-serif));
```

Comment:
`'' + $i + ''`... ?

Reply:
I need to put my font between quotes to get final output like for example `$o-theme-fonts` = `-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Noto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji", "Open Sans", sans-serif, "Source Sans Pro", sans-serif, "Raleway", sans-serif, "Noto Serif", serif, "Arvo", Times, serif, "Roboto Slab", sans-serif, "Poppins", sans-serif, "PT Sans Narrow", sans-serif, "Lato", sans-serif, "Slabo 27px", sans-serif
`

Reply:
- Don't name your font `$i`
- Is not `quote($font)` what you want ? Here you are concatening an empty string with a string with an empty string again, it makes no sense

Reply:
Thanks this is what I needed ;)